src/lib/variants.test.ts

```diff
@@ -191,41 +192,21 @@ describe('variant filters', () => {
             ...testDefault,
             userCohort: 'AllNonSupporters',
         };
-        const filter1 = inCorrectCohort('AllNonSupporters');
+        const filter1 = inCorrectCohort([
+            'PostAskPauseSingleContributors',
+            'AllNonSupporters',
+            'Everyone',
+        ]);
         const got1 = filter1.test(test1, targetingDefault);
         expect(got1).toBe(true);
 
         const test2 = {
             ...testDefault,
             userCohort: 'AllExistingSupporters',
         };
-        const filter2 = inCorrectCohort('AllExistingSupporters');
+        const filter2 = inCorrectCohort(['AllNonSupporters', 'Everyone']);
         const got2 = filter2.test(test2, targetingDefault);
-        expect(got2).toBe(true);
-
-        const test3 = {
```

Comment:
(I got rid of some of these because the `isCorrectCohort` filter is so simple now (just an array includes call) that having more than a simple test feels unnecessary.)